fnet/src/vespa/fnet/transport.h

```diff
@@ -53,6 +53,11 @@ class FNET_Transport
         : FNET_Transport(vespalib::AsyncResolver::get_shared(), vespalib::CryptoEngine::get_default(), 1) {}
     ~FNET_Transport();
 
+    size_t events_before_wakeup() const { return _events_before_wakeup; }
+    void events_before_wakeup(size_t events_before_wakeup_in) {
+        _events_before_wakeup = events_before_wakeup_in;
```

Comment:
consider protecting against 0

Reply:
I have another PR cooking that redoes config propagation. It is covered there.